### PR TITLE
COJ-335 Add content to home page

### PR DESCRIPTION
**Pull request by nbuonin**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -153,9 +153,68 @@ const Home = () => (
...
+        <div className="row">
             <Preview />
         </div>
+        <div className="row">
+            <div className="card col-md-4">
+                <div className="card-body">
+                    <h2 className="card-title">
+                       About
+                    </h2>
+                    <p className="card-text">Stats Interactives are a set of interactives used to illustrate introductory concepts in statistics. The project was built by Prof. Doru Cojoc in collaboration with the Center for Teaching and Learning.</p>
```
Comment (sdreher) [inline src/App.jsx]: Stats Interactives `is` a set?

File: src/App.jsx [hunk 2]
```diff
@@ -153,9 +153,68 @@ const Home = () => (
...
+                    </h2>
+                    <p className="card-text">
+                        Columbia University&#39;s Center for Teaching and
+                        Learning is committed to making Stats Interactices
+                        inclusive and accessible for everyone, including people
+                        with disabilities.
+                    </p>
+                    <h3>Limitations</h3>
+                    <p className="card-text">
+                        Stats Interactive graphically representats theorems to
```
Comment (sdreher) [inline src/App.jsx]: `represents`

File: src/App.jsx [hunk 3]
```diff
@@ -153,9 +153,68 @@ const Home = () => (
...
+                        convey a deeper understanding of statistical models.
+                        These graphs are not currently accessible to screen
+                        readers. If you are a Columbia University student and
+                        require assistance, please contact the university&#39;s
+                        &nbsp;<a href="https://health.columbia.edu/content/disability-services">Office of Disability Services</a>.
+                    </p>
+                    <h3>Assessment Approach</h3>
+                    <p className="card-text">
+                        We tested Stats Interactives with the following tools.
+                        Stats Interactives show minor violations of the
```
Comment (sdreher) [inline src/App.jsx]: shows?

File: src/App.jsx (hunk 1 above)
Comment (sdreher) [inline src/App.jsx]: feels more like CTL did the building...is there another way to phrase this?

Comment (nbuonin) [reply src/App.jsx]: This is kind of ambiguous. I used 'are' because 'Interactives' is plural, but 'Stats Interactives' is singular. I'll change it.

Comment (nbuonin) [reply src/App.jsx]: Yeah, I agree this is awkward.  This is how its phrased for Econ Practice, so I just cribbed that. I'll come up with something different.

**Pull request merged**